Detectors/FIT/T0/reconstruction/include/T0Reconstruction/RecPoints.h

```diff
@@ -21,7 +21,7 @@
 
 namespace o2
 {
-namespace fit
+namespace t0
```

Comment:
To be sure: what is the reason to have common V0,T0 digits (under the fit namespace) and separate RecPoints?